merlin/systems/dag/ensemble.py

```diff
@@ -72,18 +73,20 @@ def export(self, export_path, version=1):
             # max_batch_size=configs[0].max_batch_size
         )
 
-        for col_name, col_schema in self.graph.input_schema.column_schemas.items():
-            ensemble_config.input.append(
-                model_config.ModelInput(
-                    name=col_name, data_type=_convert_dtype(col_schema.dtype), dims=[-1, -1]
-                )
+        for _, col_schema in self.graph.input_schema.column_schemas.items():
+            add_model_param(
+                ensemble_config.input,
+                model_config.ModelInput,
+                col_schema,
+                col_schema.properties.get("shape", None) or compute_dims(col_schema),
```

Comment:
I'm using an ad-hoc property here since we don't yet have a `.shape` attribute on `ColumnSchema`, but I'd expect to switch over to `col_schema.shape` once that's available.